.github/actions/nm-build-docker/action.yml

```diff
@@ -27,6 +27,11 @@ runs:
              --build-arg build_type=${{ inputs.build_type }} \
              --build-arg build_version=${{ inputs.build_version }} \
              --target vllm-openai . || status=$?
+      if [ ${status} -eq 0 ] && [[ "${build_type}" = "release" ]]; then
+          echo "Also tag image for release build as latest"
+          docker image tag ghcr.io/neuralmagic/nm-vllm-openai:${{ inputs.docker_tag }} ghcr.io/neuralmagic/nm-vllm-openai:latest || ((status+=$?))
```

Comment:
It seems like the resulting command would be expanded as below; is that correct? It seems off...

```shell
docker image tag ghcr.io/neuralmagic/nm-vllm-openai:ghcr.io/neuralmagic/nm-vllm-openai:latest ghcr.io/neuralmagic/nm-vllm-openai:latest
```